app/_src/policies/meshcircuitbreaker.md

```diff
@@ -0,0 +1,305 @@
+---
+title: MeshCircuitBreaker (beta)
+---
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state, it should not be mixed
+with [CircuitBreaker](../circuit-breaker).
+{% endwarning %}
+
+This policy will look for errors in the live traffic being exchanged between our data plane proxies. It will mark a data
+proxy as unhealthy if certain conditions are met. The policy will ensure that no additional traffic can reach an
+unhealthy data plane proxy until it is healthy again.
+
+Circuit breakers - unlike active [MeshHealthChecks](/docs/{{ page.version }}/policies/meshhealthcheck/) - do not send
+additional traffic to our data plane proxies but they rather inspect the existing service traffic. They are also
+commonly used to prevent cascading failures.
+
+{% tip %}
+Like a real-world circuit breaker when the circuit is **closed** then traffic between a source and destination data
+plane proxy is allowed to freely flow through it. When it is **open** then the traffic is interrupted.
+{% endtip %}
+
+The conditions that determine when a circuit breaker is closed or open are being configured on connection limits or
+outlier detection basis. For outlier detection to open circuit breaker you can configure what we call "detectors".
+This policy provides 5 different types of detectors, and they are triggered on some deviations in the upstream service
+behavior. All detectors could coexist on the same outbound interface.
+
+Once one of the detectors has been triggered the corresponding data plane proxy is ejected from the set of the load
+balancer for a period equal to [baseEjectionTime](#outlier-detection-configuration). Every further ejection of the same data plane
+proxy will further extend the [baseEjectionTime](#outlier-detection-configuration) multiplied by the number of ejections: for example
+the 4th ejection will be lasting for a period of time of `4 * baseEjectionTime`.
+
+This policy provides **passive** checks.
+If you want to configure **active** checks, please utilize the [MeshHealthCheck](/docs/{{ page.version }}/policies/meshhealthcheckcz)
+policy.
+Data plane proxies with **passive** checks won't explicitly send requests to other data plane proxies to determine if
+target proxies are healthy or not.
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to  | from |
+|-------------------|-----------|-----|------|
+| Mesh              | ✅         | ✅   | ✅    |
+| MeshSubset        | ✅         | ❌   | ❌    |
+| MeshService       | ✅         | ✅   | ❌    |
+| MeshServiceSubset | ✅         | ❌   | ❌    |
+| MeshGatewayRoute  | ❌         | ❌   | ❌    |
+
+To learn more about the information in this table, see the [matching docs](/docs/{{ page.version }}/policies/targetref).
+
+## Configuration
+
+### Examples
```

Comment:
Usually all of the navbar entries are level 2 - https://github.com/kumahq/kuma-website/blame/18a1a3f4a97f482ef403231c5c7f42bc145c0b60/app/_src/policies/meshtrafficpermission.md#L31 and configuration has configuration options (the thing that starts [here](https://github.com/kumahq/kuma-website/pull/1214/files#diff-bf631074d8bc5d74e7ecab393d60df436a9379befbee4010871f84b639e5f0a1R195)). I don't see any reason to link in navbar to "configuration" if it's empty.

Reply:
that was just temporary, I included configuration section with the latest commit